content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,31 @@ menu:
 weight: 101
 ---
 
+## v2.0.2 General Availability [2020-11-17]
+
+### Features
+- Improvements to upgrade from 1.x to 2.x:
+    - Warning appears if auth is not enabled in 1.x, which is not an option in 2.x. For details, see [Upgrade from InfluxDB 1.x to InfluxDB 2.0](/influxdb/v2.0/upgrade/v1-to-v2/).
```

Comment:
if authentication is not enabled in 1.x (`auth-enabled = false`)

Reply:
Authentication must be enabled in 2.x? For details...

Reply:
Authentication is, by default, enabled in 2.x.  There isn't any way to disable this....